content/cli/CFY CLI Commands/executions.md

```diff
@@ -51,7 +51,8 @@ Execute a workflow on a given deployment
                         to terminate) (default: 900)
 * `-l, --include-logs / --no-logs` -   Include logs in returned events
 * `--json` -               Output events in a consumable JSON format
-* ` -t, --tenant-name TEXT`      The name of the tenant on which the execution will be executed. If unspecified, the current tenant is used.
+* ` -t, --tenant-name TEXT` -     The name of the tenant on which the execution will be executed. If unspecified, the current tenant is used.
+* `--dry-run` - Execute the workflow as a [dry-run]({{< relref "workflows/dry-run.md" >}}) so that the workflow is shown step-by-step but the workflow is not implemented and no changes are made.
```

Comment:
Maybe "the workflow is shown step-by-step" -> "the execution is shown step-by-step"? Not sure. It's a suggestion